.github/workflows/linux_test.yml

```
@@ -0,0 +1,48 @@
+name: "Linux test"
+on: [push]
```

add `pull_request`?

```suggestion
on: [push, pull_request]
```

https://help.github.com/en/articles/workflow-syntax-for-github-actions#on